scrapyd_k8s/log_handler.py

```diff
@@ -0,0 +1,161 @@
+import os
```

Comment:
This file being in the project's code root, I would expect it to be backend-agnostic. Now there are many Kubernetes-specific parts, so I would move this to `joblogs/log_handler_k8s.py` or so, to make it clear that this is specific to Kubernetes, and to make room for a Docker-based implementation (which would be possible, as far as I currently understand).